Add 23th of December 2024 Copr outage (#174)
packit/status: content/issues/2024-12-23-copr-outage.md

---
title: "Copr outage"
date: 2024-12-23T06:00:00+01:00
affected:
  - Copr
resolved: false
#resolvedWhen: 2024-12-04T16:30:00+01:00
section: issue
severity: disrupted
---

Copr services are temporarily unavailable due to a service disruption.

> In Packit you will see Copr builds stuck in a pending state.

Reviewer: ```suggestion
In Packit, this results in Copr builds stuck in a pending state.
```
